install/yamls/caa-pod.yaml

```diff
@@ -22,7 +22,7 @@ spec:
             name: peer-pods-secret
         - configMapRef:
             name: peer-pods-cm
-        image: kustomize.this/cloud-api-adaptor/image:url
+        image: quay.io/confidential-containers/cloud-api-adaptor:latest
```

Comment:
how about changing this to:
```
image: cloud-api-adaptor
```
and then uncomment it in all kustomization.yaml files to:
```
images:
- name: cloud-api-adaptor
  newName: quay.io/confidential-containers/cloud-api-adaptor
  newTag: latest
```

Reply:
Thanks for the suggestion - I've pushed a change based on this suggestion. Once people are is general agreement I'll squash the commits and move this PR out of draft.